src/Service/Users.php

```diff
@@ -158,7 +158,7 @@ public function logoutUser($authToken, array $options = []) {
         'body' => $postBody,
       ];
 
-      $apiOptions = $this->config->get('logout');
+      $apiOptions = $this->config['logout'];
```

Comment:
Please can you rename `$this->config` to something a little more descriptive? Its not the entire "config", its only operations, therefore a more logical and descriptive name would be:
`$this->openamApiOperations`

Reply:
Updated.